client/src/App.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import axios from 'axios';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    // this.state = {
+    //   name: "Ammar"
+    // }
+    this.state = {};
+    this.getData = this.getData.bind(this);
+    this.sendData = this.sendData.bind(this);
+    this.updateData = this.updateData.bind(this);
+    this.deleteData = this.deleteData.bind(this);
+  }
+
+  componentDidMount() {
+
+  }
+
+  getData() {
+    axios.get('/api/times')
+      .then(response => {
+        let data = response.data;
+        this.setState({ data });
+        console.log(this.state.data);
+      });
+  }
+
+  sendData(userData) {
+    axios.post('/api/times', userData)
+      .then(response => {
+        console.log(response);
+      })
+  }
+
+  updateData(userData) {
+    axios.put('/api/times?id=' + userData['id'] + '&name=' + userData['name'])
```

Comment:
I see that you used arguments here, which is correct but a nice way to deal with it (it's easier when we have a lot of different keys to send) is to pass a json object in the body of the call. With axios it goes like this 
```javascript
axios.put('/path/to/url', userData).then...
```
this will sent `userData`'s attribute to backend and will be available in `request.body.id` and `request.body.name` from your express request handler

Note that you can add a json body on PUT, PATCH and POST but not on GET as get is supposed to be declarative enough, if you need to specify additionnal param in a get (ex: for a search API) in that rare case using params is good `GET /api/times?searchField=name&searchFor=ammar`